shopinvader_delivery_carrier/controllers/main.py

```diff
@@ -14,4 +14,5 @@
 class InvaderController(main.InvaderController):
     @route(["/shopinvader/cart/get_delivery_methods"], methods=["GET"])
     def get_delivery_methods(self, **params):
+        # deprecated!!!
```

Comment:
wouldn't be better to log deprecation?

Reply:
I add the log only in the service. As this controller call the service, we will have twice the same message into logs.